src/predict.jl

```diff
@@ -32,6 +38,16 @@ function predict(
 
     tryload!(estimator)
 
+    if estimator.predict_raw_score && !estimator.predict_leaf_index && !estimator.predict_contrib
```

Comment:
I didn't include the same in the `predict_classes` function although it also takes the predict_type parameter that could be altered depending on the prediction params but basing on its functionality it is based on what is called the normal prediction (standard 0 .0 to 1.0 predicted probabilities)